torchci/.env.example

```diff
@@ -24,7 +24,7 @@ OPENSEARCH_REGION=
 OPENSEARCH_USERNAME=
 OPENSEARCH_PASSWORD=
 
-# Secrets for making queries to ClickHouse.  They can be found on Vercel
+# Secrets for making queries to ClickHouse.  They can be found on Keeper
```

Comment:
English is not my native, but `on Keeper` or `in Keeper`?

Reply:
eh, both are valid in this case